src/common/localization/zh-tw.ts

```diff
@@ -0,0 +1,514 @@
+import { IAppStrings } from "../strings";
+
+/**
+ * App Strings for Traditional Chinese (zh-tw)
+ */
+export const chinesetw: IAppStrings = {
+    appName: "VOTT視覺物件標記工具", // Visual Object Tagging Tool
+    common: {
+        displayName: "顯示名稱", // Display Name
+        description: "說明", // Description
+        submit: "送出", // Submit
+        cancel: "取消", // Cancel
+        save: "儲存", // Save
+        delete: "刪除", // Delete
+        provider: "提供者", // Provider
+        homePage: "首頁", // Home Page
+    },
+    titleBar: {
+        help: "說明", // Help
+        minimize: "最小化", // Minimize
+        maximize: "最大化", // Maximize
+        restore: "還原", // Restore
+        close: "關閉", // Close
+    },
+    homePage: {
+        newProject: "新專案", // New Project
+        openLocalProject: {
+            title: "打開本機專案", // Open Local Project
+        },
+        openCloudProject: {
+            title: "打開雲端專案", // Open Cloud Project
+            selectConnection: "選擇連線", // Select a Connection
+        },
+        recentProjects: "最近的專案", // Recent Projects
+        deleteProject: {
+            title: "刪除專案", // Delete Project
+            confirmation: "確定要刪除專案?", // Are you sure you want to delete project
+        },
+        importProject: {
+            title: "匯入專案", // Import Project
+            confirmation: "您確定要將專案$ {project.file.name}的設定轉換為v2格式嗎？我們建議您首先備份專案文件。",
```

Comment:
please remove the space between `$` and `{project.file.name}`.

Reply:
updated